django/contrib/auth/forms.py

```diff
@@ -469,42 +469,70 @@ class AdminPasswordChangeForm(forms.Form):
             attrs={"autocomplete": "new-password", "autofocus": True}
         ),
         strip=False,
+        required=False,  # Can be empty if usable_password is False
+        validators=[],  # Password validation happens in the check_password1() method
```

Comment:
Yes, it's both needed and related. The password validation can only happen in the form's clean method, since a password is only required or needs to be validated if `has_usable_password` is `True`.